apps/modernization-ui/src/design-system/Pagination/Pagination.tsx

```diff
@@ -22,5 +22,7 @@ export const Pagination = () => {
                 />
             </div>
         </div>
+    ) : (
+        <></>
```

Comment:
question:  What's the reason for hiding this?  shouldn't the total pages always be there?

Reply:
I saw this when I was making a search and there were no results I was seeing infinite pages displaying. So thats why I added a condition on that total number of results

Reply:
ahh ok, that should hopefully go away with the No results story